### PR TITLE
duckduck

### PR DESCRIPTION
**Pull request by dmitry-ermakov**

### CODE REVIEW

File: tests/DDG.spec.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const { test, expect } = require('@playwright/test');
+
+/*test('basic test', async ({ page }) => {
+  await page.goto('https://playwright.dev/');
+  //await page.pause();
+  const title = page.locator('.navbar__inner .navbar__title');
+  await expect(title).toHaveText('Playwright');
+});*/
+
+test.only('check logo', async ({ page }) => {
```
Comment (jevgenijusmarinuskinas) [inline tests/DDG.spec.js:10]: We have only single test, I would suggesr not using test.only here. Also, test name should be meaningful and reflect what it is doing, e.g. 'Checks that duckduckGo page can be opened'

File: tests/DDG.spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  await page.goto('https://duckduckgo.com/');
+  const isLogoVisible = await page.isVisible('#logo_homepage_link');
+  //const logo = page.locator('#logo_homepage_link');
+  await expect(isLogoVisible).toBe(true);
+});
```
Comment (jevgenijusmarinuskinas) [inline tests/DDG.spec.js:15]: Please add a new line at the end of file

**Review by jevgenijusmarinuskinas — APPROVED** [review]

File: tests/foo.spec.js [hunk 3]
```diff
@@ -1,7 +0,0 @@
-const { test, expect } = require('@playwright/test');
```
Comment (ernestas-zekas) [inline tests/foo.spec.js:1]: This file can be deleted.

File: tests/DDG.spec.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  await page.goto('https://playwright.dev/');
+  //await page.pause();
+  const title = page.locator('.navbar__inner .navbar__title');
+  await expect(title).toHaveText('Playwright');
+});*/
+
+test.only('check logo', async ({ page }) => {
+  await page.goto('https://duckduckgo.com/');
+  const isLogoVisible = await page.isVisible('#logo_homepage_link');
+  //const logo = page.locator('#logo_homepage_link');
```
Comment (ernestas-zekas) [inline tests/DDG.spec.js:13]: Commented out code can be deleted.

**Review by ernestas-zekas — APPROVED** [review]

**Pull request merged**